HeavyIonsAnalysis/JetAnalysis/python/hiFJRhoAnalyzer_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+hiFJRhoAnalyzer = cms.EDAnalyzer('HiFJRhoAnalyzer',
+                                 etaMap    = cms.InputTag('hiFJRhoProducer','mapEtaEdges','HiForest'),
+                                 rho       = cms.InputTag('hiFJRhoProducer','mapToRho'),
```

Comment:
You usually don't need to specify the third argument ("HiForest" here) unless you want to pick up the results from a previous process (usually "HLT", for the trigger results).